db/migrate/20190716173854_add_secure_to_uploads.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class AddSecureToUploads < ActiveRecord::Migration[5.2]
+  def up
+    add_column :uploads, :secure, :boolean, default: false, null: false
+
+    Upload.find_each do |upload|
```

Comment:
We need to stick to manual SQL in migrations, because the fields in objects change over time and it's too easy to break old migrations who count on a particular table structure.